packages/auth-client/src/idleManager.ts

```diff
@@ -114,21 +124,25 @@ class IdleManager {
    * @param {IdleCB} callback function to be called when user goes idle
    */
   public registerCallback(callback: IdleCB): void {
+    if (this._usingDefaultCb) {
+      this.callbacks = [];
+    }
```

Comment:
Would it be possible to add a test that the local storage is cleared AND the refresh happens when the timeout occurs?

Reply:
This test will live in the index.test.ts file, since the logout functionality isn't directly a concern of the idleManager itself